requirements-base.txt

```diff
@@ -1,104 +1,104 @@
-beautifulsoup4>=4.7.1
```

Comment:
We're freezing the deps but we're not yet using pip tools to generate it, correct?

Reply:
Uhh, you're commenting/reviewing an intermediate commit where I made a mistake in resolving a merge, `>=` pins are intended.